src/content/docs/es/tutorial/4-layouts/index.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 type: tutorial
 unitTitle: Ahorra tiempo y energía con plantiallas de página reutilizables
-title: 'Regístrate: Unidad 4 - Plantillas'
+title: 'Check in: Unidad 4 - Plantillas'
```

Comment:
Check-in es Registro en español, hay anglisismos que si pueden traducirse.

Reply:
Esta corrección la hice para estandarizar los títulos de las páginas, porque en el resto de unidades del tutorial está como "Check in".

Reply:
Ya revisé y si tienes razón.